recipes/deepchem/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "deepchem" %}
+{% set version = "2.3.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/deepchem/deepchem/archive/{{ version }}.tar.gz
+  sha256: eec0d4dfa23e234d26d171ef20c62a699052a1bc230bbace341047e0c2673030
+
+build:
+  noarch: python
+  number: 0
+  script: "PBR_VERSION=${PKG_VERSION} && export PBR_VERSION && {{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv"
```

Comment:
```suggestion
  script: "PBR_VERSION=${PKG_VERSION} && export PBR_VERSION && {{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv"
  skip: True   # [win]
```

Reply:
It seems xgboost is not available for windows, could you also commit this line @hadim?